tests/codegen_testcases/solidity/constant_folding.sol

```diff
@@ -2,9 +2,40 @@
 
 contract CodeWithJD {
     mapping(address => uint256) balances;
+
     // BEGIN-CHECK: CodeWithJD::CodeWithJD::function::totalSupply
     function totalSupply() public view returns (uint256) {
         // CHECK: load storage slot(hex"b55fba97e5495840b2400ab391e4362b96f1173f44a58442cdd3f776b62832ad") ty:uint256
         return balances[address(0x00)];
     }
 }
+
+contract Enum {
+    enum Lender {
+        USDT,
+        USDC,
+        DAI
+    }
+    mapping(Lender => address) lenders;
+
+    constructor(
```

Comment:
I think you need to add the `BEGIN-CHECK` here, so the tester code can split each function test.